library/Icingadb/Web/Controller.php

```diff
@@ -367,7 +367,7 @@ public function createViewModeSwitcher(
         LimitControl $limitControl,
         bool $verticalPagination = false
     ): ViewModeSwitcher {
-        $viewModeSwitcher = new ViewModeSwitcher();
+        $viewModeSwitcher = new ViewModeSwitcher(Url::fromRequest());
```

Comment:
It doesn't need to parse the url to know the current view mode, it's already passed by `populate`, just not accessed internally.